src/components/cards/OpRewardsCard.tsx

```diff
@@ -1,75 +1,83 @@
 import React from 'react';
-import { GenericCard } from '@/components/cards/GenericCard';
-import { Box, Cover, globalColors, Stack, Text } from '@/design-system';
+import { GenericCard, Gradient } from './GenericCard';
+import {
+  AccentColorProvider,
+  Box,
+  Cover,
+  globalColors,
+  Stack,
+  Text,
+} from '@/design-system';
 import { ButtonPressAnimation } from '@/components/animations';
-import { Image, StyleSheet } from 'react-native';
+import { Image } from 'react-native';
 import OpRewardsCardBackgroundImage from '../../assets/opRewardsCardBackgroundImage.png';
-import { useTheme } from '@/theme';
 import * as i18n from '@/languages';
 import { useNavigation } from '@/navigation';
 import Routes from '@/navigation/routesNames';
+import { colors } from '@/styles';
```

Comment:
@jkadamczyk this was the color fix I DM'ed you about

Reply:
Thank you!